### ISSUE TITLE
No Appropriate Version Found

### ISSUE THREAD

**Issue by Caz63** [alexkurowski/solo-toolkit#15]
Just now I tried to install your plugin via Obsidian "community plugins." Pressing the Install button produced the message "no appropriate version found"  and nothing happened ...
![No appropriate version found](https://github.com/user-attachments/assets/d0bd8c3f-8ef3-4c2b-891a-f1e7922cd5ee)

**Comment by alexkurowski** [comment]
Minimum supported Obsidian version is 1.5.11 (https://github.com/alexkurowski/solo-toolkit/blob/d336e6fc11deb53be0031ca92af59df2bc298208/manifest.json#L5)

You might be able to install this plugin manually though, or update your Obsidian and try again.

**Comment by Caz63** [comment]
Yes, I indeed managed to install the plugin manually, thank you!  It works perfectly!
The reason why I am using version 1.5.8 is because in the next version somebody screwed up translucency and thus wrecked my interface. I do not like that! ☹
